src/Explorer/Panes/SettingsPaneR/index.tsx

```diff
@@ -0,0 +1,433 @@
+import { PrimaryButton } from "office-ui-fabric-react";
+import React, { FunctionComponent, KeyboardEvent, useEffect, useLayoutEffect, useRef, useState } from "react";
+import * as Constants from "../../../Common/Constants";
+import { configContext } from "../../../ConfigContext";
+import { LocalStorageUtility, StorageKey } from "../../../Shared/StorageUtility";
+import * as StringUtility from "../../../Shared/StringUtility";
+import * as NotificationConsoleUtils from "../../../Utils/NotificationConsoleUtils";
+import Explorer from "../../Explorer";
+
+export interface SettingsPaneProps {
+  explorer: Explorer;
+  closePanel: () => void;
+}
+
+export const SettingsPaneR: FunctionComponent<SettingsPaneProps> = ({ explorer, closePanel }: SettingsPaneProps) => {
```

Comment:
Rename this to just `SettingsPane`. Remove the old knockout based settings pane

Reply:
done